vk_api/audio.py

```diff
@@ -221,6 +221,24 @@ def get_audio_by_id(self, owner_id, audio_id):
         link = bs.select_one('.ai_body input[type=hidden]').attrs['value']
         return decode_audio_url(link, self.user_id)
 
+    def get_wall_audio(self, owner_id, wall_id):
```

Comment:
ну post тут больше подходит, wall это если бы со стены

Reply:
Тогда я верну прежнее имя метода:
```suggestion
    def get_post_audio(self, owner_id, post_id):
```